tools/documentation/deployment-checklist.md

```diff
@@ -0,0 +1,116 @@
+
+1. Create new branch `release/#.#.#` from `develop`.
+1. Open a pull request for `release/#.#.#` into `master`.
+    - Include a title of the version number, eg 'v5.2.0'.
+    - Include a description of 'Deploy v5.2.0'.
+1. On Heroku, change staging app to point to this new release branch.
+1. Broadcast publishing of testing app, and seek bug reports or feedback.
+1. Merge any applicable bugfix branches into `release/#.#.#`.
+    - And then on `develop`, run `git merge release/#.#.#` (results in a FF) and push.
+
+_Only bugfix branches should be merged (to `release/#.#.# --> develop`) during this phase._
+
+Upon completion of the testing phase, conduct the release procedure (outlined below).
+
+---
+
+## Development Procedures
+1. Merge all feature and bugfix branches:
+    - Use non-FF merges into `develop`, using either of the below commands:
+        - `git pull origin feature/###` (merges local version of branch)
+        - `git merge --no-ff feature/###` (merges latest version of upstream branch)
+    - Set/amend the merge commit's message to take the following form:
+        - Include summary of `Merge [feature/bugfix]/branchName (#PRNumber)`.
+        - Include description that very briefly explains the purpose of that branch.
+1. Merge all hotfix branches:
+    - Use non-FF merges into `master`, using either of the below commands:
+        - `git pull origin bugfix/###` (merges local version of branch)
+        - `git merge --no-ff bugfix/###` (merges latest version of upstream branch)
+    - Set/amend the merge commit's message to take the following form:
+        - Include summary of `Merge hotfix/branchName (#PRNumber)`.
+        - Include description that very briefly explains the purpose of that branch.
+    - Merge `master` into `develop` with `git merge master`.
+
+### Release Procedures
+1. Attempt to merge `release` into `develop` with `git merge release/#.#.#`.
+    - By design, `develop` should contain no changes, resulting in git replying that develop is `Already up-to-date.`.
+1. Checkout `release/#.#.#`.
+1. Create commit `ARCH - Finalize CHANGELOG and set version number for v#.#.# release`.
+    - Set new release's version number:
+        - in `package.json`.
+        - in `VERSION` within `App.js`.
+    - Clean up changelog file.
+        - Remove the extra empty lines in each section.
+        - Update current sprint's version number and release date.
+1. Merge `release/#.#.#` into `master` with `git merge --no-ff release/#.#.#`.
+    - Include summary of `Deploy v5.2.0 (#123)`.
+1. Ensure all tests are passing on `develop` and `release/#.#.#` and `master`.
```

Comment:
This is the only spot I have some disagreement.

- Immediately after merging `release` into `master` tests should be verified.  
- When all tests are green, a new tag can be created 
-  `master` and the new tag can both be pushed.
- Then `develop` can be updated

doing it this way means you don't have to checkout `master` twice and has more of a _top down_ feel to it where you get everything ready to go into `master` and once it is, you update everything else down the tree.